app/src/components/MovieInfo.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getMovies } from '../actions';
+import './MovieInfo.css';
+
+class MovieInfo extends React.Component {
+  componentDidMount() {
+    this.props.getMovies();
+  }
+  render() {
+    const { movies } = this.props;
+    const id = this.props.match.params.id;
+    if (movies[id] === undefined) return null;
```

Comment:
Here, since you're only checking if `movies[id] === undefined`, you'll get a runtime error if `movies[id]` is `null`. Would probably be better to change it to `if (!movies[id]) return null;`.